static/styles/zulip.scss

```diff
@@ -546,6 +546,69 @@ strong {
     color: inherit;
 }
 
+// .dropdown-menu from v2.3.2
+// + https://github.com/zulip/zulip/commit/7a3a3be7e547d3e8f0ed00820835104867f2433d
+// basic idea of this fix is to remove decorations from :hover and apply them only
+// on :focus.
```

Comment:
Can't we just use a `!important` override for just the one or two properties we want to override?  That seems likely to be a lot cleaner.

Reply:
I would prefer to keep the whole of `.dropdown-menu` css here to avoid future conflicts when upgrading to v3.x. Most of the fields here are important, I can trim down 1 or 2 but I don't think it would really make a difference.